### DISCUSSION TITLE
Android 14

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#277] [Q&A] [answered]
[Android 14](https://developer.android.com/about/versions/14) compatibility.

**Comment by devemux86** [comment, marked as answer]
Implemented in [Cruiser 3.0.30](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Reply by DolomitesTom** [reply]
Does anyone already have Android 14 :-) ?

**Reply by devemux86** [reply]
Android Emulator. :slightly_smiling_face:
